client/src/components/PriceVolumeChart.jsx

```diff
@@ -13,18 +14,24 @@ class PriceVolumeChart extends React.Component {
             heighest: 0,
             averagePrice: 0,
             currentPrice: 0,
-            difference:0
+            difference:0,
+            currentPriceIndex:0,
+            averagePriceIndex:0,
         } 
+        this.findBarGraphIndex = this.findBarGraphIndex.bind(this);
+        this.indexInRange = this.indexInRange.bind(this);
+        this.barWidth = 11.457627118644067;
     }
+
     componentDidMount(){
         this.handleFetch();
     }
 
     handleFetch(id = '5bd277855f59f4e021dd7a62'){
```

Comment:
We should talk about the id's we are using as a group, as all ids that are generated in MongoDB are unique.

Reply:
Yes. As we discussed earlier, I would refactor my data schema.